### PR TITLE
Issue 1835

### PR DESCRIPTION
**Pull request by jkburges**
I have to admit, I'm not overly happy with 642a4ae, but it does seems to work ok for me.

### CODE REVIEW

**Comment by danfruehauf** [comment]
On that branch, in chrome I get the following when trying to add a collection:
```
Uncaught TypeError: Cannot read property 'added' of undefined
```

File: web-app/js/portal/PortalEvents.js [hunk 1]
```diff
@@ -3,8 +3,6 @@ var PORTAL_EVENTS = {
     SELECTED_LAYER_CHANGED:        "selectedLayerChanged",
     BASE_LAYER_CHANGED:            "baseLayerChanged",
     LAYER_REMOVED:                 "layerRemoved",
-    LAYER_LOADING_START:           "layerLoadingStart",
```
Comment (danfruehauf) [inline web-app/js/portal/PortalEvents.js]: While you're at it, also those can be removed:
 * `BEFORE_SELECTED_LAYER_CHANGED`
 * `LOAD_SNAPSHOT`

File: web-app/js/portal/ui/openlayers/MapOptions.js [hunk 2]
```diff
@@ -50,6 +50,7 @@ Portal.ui.openlayers.MapOptions = Ext.extend(Object, {
             this.clickControl
         ];
 
+        this.tileManager = null;
```
Comment (danfruehauf) [inline web-app/js/portal/ui/openlayers/MapOptions.js]: Why is this needed here?

File: web-app/js/portal/details/SubsetItemsWrapperPanel.js [hunk 3]
```diff
@@ -74,6 +89,7 @@ Portal.details.SubsetItemsWrapperPanel = Ext.extend(Ext.Panel, {
         this.spinnerToolItem = {
             id: 'spinnerToolItem',
             styles: 'fa-spin fa-spinner',
+            hidden: !layer.loading,
```
Comment (danfruehauf) [inline web-app/js/portal/details/SubsetItemsWrapperPanel.js]: Would it be safer to have `layer.loading !== false`? Just not sure what behaviour you want when `layer.loading` is either `undefined` or `null`.

File: web-app/js/portal/PortalEvents.js (hunk 1 above)
Comment (jkburges) [inline web-app/js/portal/PortalEvents.js]: I'll do this on another PR.

Comment (danfruehauf) [inline web-app/js/portal/PortalEvents.js]: Fair.

File: web-app/js/portal/ui/openlayers/MapOptions.js (hunk 2 above)
Comment (jkburges) [inline web-app/js/portal/ui/openlayers/MapOptions.js]: It "fixes" the problem that you mention here: https://github.com/aodn/aodn-portal/commit/63472c212a1c2cdfb4cffb6459fd7e4919cb91e2

Along with this, and having our override removed, panning + layer events works.  Like I say above, I'm not very happy with this, hopefully I haven't broken anything.

File: web-app/js/portal/details/SubsetItemsWrapperPanel.js (hunk 3 above)
Comment (jkburges) [inline web-app/js/portal/details/SubsetItemsWrapperPanel.js]: If `undefined` or `null`, then the spinner will be hidden, which I think is correct.  I prefer this style (of "truthiness") but it's just a preference.

**Comment by jkburges** [comment]
The `Cannot read property 'added' of undefined` error has now been fixed.  It was a rebase gone wrong, and me not testing properly afterwards :crying_cat_face:

File: web-app/js/portal/PortalEvents.js (hunk 1 above)
Comment (jkburges) [inline web-app/js/portal/PortalEvents.js]: See: https://github.com/aodn/aodn-portal/pull/1856
